operations/commit_queue.go

```diff
@@ -392,7 +392,7 @@ func (p *moduleParams) addModule(ac *legacyClient, rc *legacyClient) error {
 	if err != nil {
 		return errors.WithStack(err)
 	}
-
+	fmt.Println("Module updated.")
```

Comment:
I think we've been printing to the console only through grip, though I'm not 100 percent clear why.